### PR TITLE
We need to force a size changed event _after_ layout so that the chrome web view has the correct size.

### PR DESCRIPTION
**Pull request by samgiles**
@wilsonpage r? (Also check it works for you now :p)

Might also be worth checking this doesn't mess up the webview you were previously using.

### CODE REVIEW

File: android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebView.java [hunk 1]
```diff
@@ -61,18 +68,39 @@ public void cleanupCallbacksAndDestroy() {
 
     @Override
     protected void onLayout(boolean changed, int left, int top, int right, int bottom) {
-        super.onLayout(changed, left, top, right, bottom);
         Log.d(TAG, "on layout");
         setLayout();
+
+        super.onLayout(changed, left, top, right, bottom);
```
Comment (wilsonpage) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebView.java:70]: Any reason this isn't called before setLayout?

Comment (samgiles) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebView.java:70]: No reason - I changed.

**Comment by samgiles** [comment]
Sorry, I noticed loads of bad stuff after I r?'d it. 😬

**Comment by wilsonpage** [comment]
Working on my device!!! \o/ r+

**Comment by samgiles** [comment]
The travis job's output says they finished successfully... So I'm gonna merge.

**Pull request merged**